bsp/boards/radio.h

```diff
@@ -66,7 +66,7 @@ void     radio_setFrequency(uint8_t frequency);
 void     radio_rfOn(void);
 void     radio_rfOff(void);
 // TX
-void     radio_loadPacket(uint8_t* packet, uint8_t len);
+void     radio_loadPacket(uint8_t* packet, uint16_t len);

```

Comment:
In fact to be cross platform data types should be size_t for numbers,  uintptr_t for address, void for pointers...

----- Mensagem Original -----
De: "Thomas Watteyne" <notifications@github.com>
Enviado: ‎17/‎08/‎2016 18:16
Para: "openwsn-berkeley/openwsn-fw" <openwsn-fw@noreply.github.com>
Assunto: Re: [openwsn-berkeley/openwsn-fw] FW-527. Port CC1200dk (#280)

In bsp/boards/radio.h:
> @@ -66,7 +66,7 @@ void     radio_setFrequency(uint8_t frequency);
>  void     radio_rfOn(void);
>  void     radio_rfOff(void);
>  // TX
> -void     radio_loadPacket(uint8_t* packet, uint8_t len);
> +void     radio_loadPacket(uint8_t* packet, uint16_t len);
If you change the signature, you need to change the BSP of all boards. You must have received an e-mail from Travis telling you the builds failed. It's a small change, and I suggest you fix the BSPs of the different boards, until Travis passes again.
—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or mute the thread.